templates/cli/lib/commands/deploy.js.twig

```diff
@@ -215,17 +215,7 @@ const deployFunction = async ({ functionId, all, yes } = {}) => {
                             parseOutput: false
                         });
                     }));
-                }
-
-                // Deploy local variables
```

Comment:
Sorry I was probably confusing in my last comment.
The logic here is for when a function already exists, and it does:
1. Ask if you are okay loosing variables
2. Delete existing varuables
3. Create variables from local config

So this part has to stay here.

Reply:
The addition you made inside catch() is now correct by the way. That looks great 😇

Reply:
Why should we delete existing variables. Surely we should only delete variables that are conflicting. That way if there are some variables that exist in the console that were created manually, we won't loose them.

Reply:
The problem with that is that if you then tried to deploy the function to a different Appwrite instance, the function would be depending on variables that don't exist in the configuration

Reply:
Resolving with my commit.